x-pack/plugins/enterprise_search/public/applications/workplace_search/utils/read_uploaded_file_as_base64.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { readUploadedFileAsBase64 } from './';
+
+describe('readUploadedFileAsBase64', () => {
+  it('reads a file as data URL', async () => {
+    const file = new File(['a mock file'], 'mockFile.png', { type: 'img/png' });
+    const text = await readUploadedFileAsBase64(file);
+    expect(text).toEqual('data:image/png;base64,YSBtb2NrIGZpbGU=');
```

Comment:
This is fixed in a future commit https://github.com/elastic/kibana/pull/108235/commits/7524142847f26009beddeec12db40b4cae0866a5